src/main.rs

```diff
@@ -28,7 +28,12 @@ fn main() {
         CompileParameters::parse(args);
     match compile_parameters {
         Ok(cp) => {
-            if let Err(msg) = build_with_params(cp) {
+            if cp.commands.is_some() {
+                if let Err(msg) = build_with_subcommand(cp) {
+                    println!("Error: {:?}", msg);
+                    std::process::exit(1);
+                }
+            } else if let Err(msg) = build_with_params(cp) {
                 println!("Error: {:?}", msg);
```

Comment:
I mentioned eprintln in the previous commit, looks like we also didn't use it, if you  end up fixing it, also fix it here please